source/agora/node/Ledger.d

```diff
@@ -202,6 +202,34 @@ public class Ledger
 
         return null;
     }
+
+
+    /***************************************************************************
+
+        Get the array of hashs the merkle branch.
+
+        Params:
+            block_height = the starting block height to begin retrieval from
+            hash         = transaction hash
+
+        Returns:
+            the array of hashs the merkle branch
+
+    ***************************************************************************/
+
+    public Hash[] getMerkleBranch (ulong block_height, Hash hash) @safe
```

Comment:
This function should be unittested (not network tests but tests inside this module).

Reply:
I mean in addition to the network tests.

Reply:
Okay